downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md

```diff
@@ -68,8 +101,66 @@ To view bug fixes, see the [GitHub milestone for 2201.8.0 (Swan Lake)](https://g
 
 ### New features
 
+#### New Runtime Java APIs
+
+- Introduced the `getImpliedType()` API in the `io.ballerina.runtime.api.utils.TypeUtils` class to retrieve the referred type if a given type is a type-reference type or to retrieve the effective type if the given type is an intersection type.
```

Comment:
```suggestion
- Introduced the `getImpliedType()` API in the `io.ballerina.runtime.api.utils.TypeUtils` class to retrieve the referred type if a given type is a type-reference type, or to retrieve the effective type if the given type is an intersection type.
```

Reply:
Is this phrased correctly? If it is a type reference type where the referred type is an intersection type, don't we get the effective type? From this, it reads like the intersection type is returned.